.gitmodules

```diff
@@ -37,4 +37,10 @@
 [submodule "packages/contracts/lib/chainlink-brownie-contracts"]
 	path = packages/contracts/lib/chainlink-brownie-contracts
 	url = https://github.com/smartcontractkit/chainlink-brownie-contracts
-	branch= main
\ No newline at end of file
+	branch= main
+[submodule "packages/contracts/lib/aave-v3-core"]
+	path = packages/contracts/lib/aave-v3-core
+	url = https://github.com/aave/aave-v3-core
+[submodule "packages/contracts/lib/aave-v3-periphery"]
+	path = packages/contracts/lib/aave-v3-periphery
+	url = https://github.com/aave/aave-v3-periphery
```

Comment:
Could you specify the branches used? ([example](https://github.com/ubiquity/ubiquity-dollar/blob/02bbe8f425bca7d720e4d0599a86eb28b25a77f4/.gitmodules#L28))